lib/cictl/solr_client.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require "rsolr"
+
+module CICTL
+  class SolrClient
+    attr_reader :solr
```

Comment:
Might want to set this up as a SimpleDelegator so everything not defined here gets passed along to the rsolr instance.